_src/pattern-library/sass/components/_layouts.scss

```diff
@@ -0,0 +1,700 @@
+%layout-view {
+    @include container($susy-view);
+}
+
+// CASE: View-based layout with flush columns
+%layout-view-flush {
+    @include container($susy-view-flush);
+}
+
+// CASE: View-based layout with reversed columns
+%layout-view-reversed {
+    @include container($susy-view-reversed);
+}
+
+
+// basic layout column
+%layout-column {
+    @include span(full); // plan for mobile first
+}
+
+// ----------------------------
+// #CANNED - IN VIEW
+// ----------------------------
+
+// basic layout
+.layout {
+    @extend %layout-default;
+}
+
+// basic layout - flush
+.layout-flush {
+    @extend %layout-flush;
+}
+
+// basic columns
+.layout-col {
+    @extend %layout-column;
+}
+
+// Full Layout
+.layout-full {
+    @extend %layout-default;
+
+    .layout-col-a,
+    .layout-col-b,
+    .layout-col-c,
+    .layout-col-d {
+        @include span(full);
+    }
+}
+
+// 1/4 + 3/4 Layout
+.layout-1q3q {
+
+    // CASE: standard layout
+    &.layout {
+        @extend %layout-default;
+        @include susy-breakpoint($bp-screen-md, $susy-default) {
+
+            .layout-col-a {
+                @include span(3 of 12 first);
+            }
+
+            .layout-col-b {
+                @include span(9 of 12 last);
+            }
+        }
+    }
+
+    // CASE: flush layout
+    &.layout-flush {
+        @extend %layout-flush;
+        @include susy-breakpoint($bp-screen-md, $susy-flush) {
+
+            .layout-col-a {
+                @include span(3 of 12 first);
+            }
+
+            .layout-col-b {
+                @include span(9 of 12 last);
+            }
+        }
+    }
+
+    // CASE: reversed layout
+    &.layout-reversed {
+        @extend %layout-default;
+        @include susy-breakpoint($bp-screen-md, $susy-reversed) {
+
+            .layout-col-a {
+                @include span(3 of 12 first);
+            }
+
+            .layout-col-b {
+                @include span(9 of 12 last);
+            }
+        }
+    }
+}
+
+// 1/3 + 1/3 + 1/3 Layout
+.layout-1t1t1t {
+
+    // CASE: standard layout
+    &.layout {
+        @extend %layout-default;
+        @include susy-breakpoint($bp-screen-md, $susy-default) {
+
+            .layout-col-a {
+                @include span(4 of 12 first);
+            }
+
+            .layout-col-b {
+                @include span(4 of 12);
+            }
+
+            .layout-col-c {
+                @include span(4 of 12 last);
+            }
+        }
+    }
+
+    // CASE: flush layout
+    &.layout-flush {
+        @extend %layout-flush;
+        @include susy-breakpoint($bp-screen-md, $susy-flush) {
+
+            .layout-col-a {
+                @include span(4 of 12 first);
+            }
+
+            .layout-col-b {
+                @include span(4 of 12);
+            }
+
+            .layout-col-c {
+                @include span(4 of 12 last);
+            }
+        }
+    }
+
+    // CASE: reversed layout
+    &.layout-reversed {
+        @extend %layout-default;
```

Comment:
NOTE: This whole set of definitions (used in all layouts) could be made more efficient through a mixin or function, but that seemed like a clean up/optimization task.